### PR TITLE
Create a facility 

### PR DESCRIPTION
**Pull request by casperjureskog**
**PT Story:** https://www.pivotaltracker.com/story/show/143869689

**Changes proposed in this PR:**

* Adds facility view
* Adds facility controller
* Adds facility form

**What I have learned working on this feature:**
To add checkbox to form and change value to true or false

**Screenshots:**
screen shot 2017-04-20 at 14 16 36

### CODE REVIEW

File: app/controllers/facilities_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class FacilitiesController < ApplicationController
+  def new
+    @facilities = Facility.new
```
Comment (tochman) [inline app/controllers/facilities_controller.rb]: `@facility`

File: app/controllers/facilities_controller.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  end
+
+  def create
+    @facility = Facility.new facility_params
+    if @facility.save
+      flash[:notice] = "New facility created!"
+      redirect_back(fallback_location: new_facility_path)
```
Comment (tochman) [inline app/controllers/facilities_controller.rb:10]: Why did you choose to redirect back to the same view?

File: app/views/facilities/new.html.haml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+%h3 Create a new facility
+= form_for (@facilities) do |f|
+  .field
+    = f.label :name
+    %br/
+    = f.text_field :name
+  .field
+    = f.label :description
+    %br/
+    = f.text_field :description
```
Comment (tochman) [inline app/views/facilities/new.html.haml]: I would suggest choosing a text area field for description.

File: app/views/facilities/new.html.haml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  .field
+    = f.label :rules
+    %br/
+    = f.text_field :rules
+  .field
+    = f.label 'Available'
+    %br/
+    = f.check_box(:status,{}, true , false)
```
Comment (tochman) [inline app/views/facilities/new.html.haml]: add spacing after comma.

File: app/views/facilities/new.html.haml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  .actions
+    = f.submit "Submit"
+
+- if flash[:notice]
+  .notice= flash[:notice]
```
Comment (tochman) [inline app/views/facilities/new.html.haml]: Not sure about adding flash display to a template. Please consider adding it to the mail application layout.

**Review by tochman — CHANGES_REQUESTED** [review]

File: app/controllers/facilities_controller.rb (hunk 2 above)
Comment (Scharrre97) [reply app/controllers/facilities_controller.rb:10]: we keep it like this for a while, because we have not create a index root view yet.

**Review by tochman — APPROVED** [review]

**Pull request merged**